mbtools/utils.py

```diff
@@ -56,12 +65,29 @@ def parse_question_bank_for_html(mbz_dir, ids=None):
     return html
 
 
-def find_references_containing(content_etree, src_content):
+def find_references_containing(content_etree, src_content=None):
```

Comment:
I think you're only modifying the signature of this function for test code which is a bit of an anti-pattern. I'd revert this to its original form and instead just use the xpath query you need in your test code.

Reply:
What do you mean by anti-pattern? I'm a little confused, I figured functionality to find all `src` resources would be a helpful addition for our testing patterns